### PR TITLE
Update projects page and Applications page to bootstrap 5

### PR DESCRIPTION
**Pull request by aanil**
Not really sure how the Applications page is supposed to work, so it is mostly as is.
Also removed unused cronjobs.js file

Also up on stage!

### CODE REVIEW

File: run_dir/design/application.html [hunk 1]
```diff
@@ -11,14 +11,12 @@
 <div class="container-fluid">
   <h2 id="page_title">Projects with application <em>{{ application }}</em></h2><br>
   <ul id="project_list">
-		<tbody></tbody>
-  </table>
```
Comment (alneberg) [inline run_dir/design/application.html:15]: Weird that this </table> thing was left here.

**Review by sofiahaglund — COMMENTED** [review]
The applications page in production doesn't show any chart for me, so I think you fixed it! It looks good to me.

The projects page as well. I'll leave the code reviewing to Johannes, but everything looks good and seems to work as it should :-)

File: run_dir/design/application.html [hunk 2]
```diff
@@ -11,14 +11,12 @@
...
 </div>
 
 <script>
 $.getJSON("/api/v1/application/{{ application }}", function(data) {
   $.each(data, function(k, project) {
-    $("#project_list").append('<li><a href="/projects/' + project + '">' + project + '</a></li>')
+    $("#project_list").append('<li><a class="text-decoration-none" href="/projects/' + project + '">' + project + '</a></li>')
   })
 })
```
Comment (alneberg) [inline run_dir/design/application.html:21]: I'm pretty sure this page is not used anywhere, but it seems to work now.

File: .gitmodules [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[submodule "run_dir/static/nvd3"]
+	path = run_dir/static/nvd3
+	url = https://github.com/novus/nvd3.git
```
Comment (alneberg) [inline .gitmodules:3]: Should we add something to the internal docs how to pull this properly when deploying?

File: run_dir/design/projects.html [hunk 4]
```diff
@@ -12,164 +12,175 @@
...
-                              <li><b>Reception Control</b>: Have Open Date, but not Queue or Close Dates.</li>
+                              <li><b>Pending</b>: Does not have Open, Queue or Close Dates.</li>
+                              <li><b>Reception Control</b>: Has Open Date, but not Queue or Close Dates.</li>
                               <li><b>Needs Review</b>: May have Open and Queue Dates, but not Close Date.</li>
-                              <li><b>Ongoing</b>: Have Open and Queue Date, but not Close Date.</li>
-                              <li><b>Open</b>: Have Open Date.</li>
-                              <li><b>Closed</b>: Have Close Date. May have Open and Queue dates.</li>
-                              <li><b>Aborted</b>: Have Aborted date.</li>
+                              <li><b>Ongoing</b>: Has Open and Queue Dates, but not Close Date.</li>
+                              <li><b>Open</b>: Has Open Date.</li>
```
Comment (alneberg) [inline run_dir/design/projects.html:64]: Should it say that it does not have close or aborted date as well? If that is correct

File: run_dir/design/projects.html [hunk 5]
```diff
@@ -12,164 +12,175 @@
...
+                    <label for="statusbtnBar1" class="fw-bold pb-2">Choose Status <button type="button" class="btn btn-xs btn-outline-secondary" id="filterInfo">
+                      <span class="fa fa-info-circle"></span></button></label>
+                      <div id='displayInfo' style="display:none;" class="alert alert-info mt-1" role="alert">
                         <div class="text-left">
+                          <span type="button" class="btn-close float-right" aria-label="Close"></span>
                             The Project status/Dates relationships are as follows:
                             <ul>
-                              <li><b>Pending</b>: Do not have Open, Queue or Close Dates.</li>
-                              <li><b>Reception Control</b>: Have Open Date, but not Queue or Close Dates.</li>
+                              <li><b>Pending</b>: Does not have Open, Queue or Close Dates.</li>
```
Comment (alneberg) [inline run_dir/design/projects.html:60]: How about aborted?

File: run_dir/design/application.html (hunk 2 above)
Comment (aanil) [reply run_dir/design/application.html:21]: I think it navigates to this page when you click on the links in the table in the Applications page.

File: .gitmodules (hunk 3 above)
Comment (aanil) [reply .gitmodules:3]: It will be a one time thing, so maybe in the section for setting up?

File: run_dir/design/projects.html (hunk 4 above)
Comment (aanil) [reply run_dir/design/projects.html:64]: This is mostly for the date filtering below, where we don't have a field for filtering the Aborted date.

File: .gitmodules (hunk 3 above)
Comment (aanil) [reply .gitmodules:3]: This seems to already be in the readme.

**Review by alneberg — APPROVED** [review]
Looks good! Great job converting the 1k lines of javascript!

**Pull request merged**